include/wups/config/WUPSConfigItemBoolean.h

```diff
@@ -89,12 +89,12 @@ WUPSConfigItemBoolean_AddToCategoryEx(WUPSConfigCategoryHandle cat,
 #include "WUPSConfigItem.h"
 #include <optional>
 #include <stdexcept>
-#include <string>
+#include <string_view>
 #include <wups/config_api.h>
 
 class WUPSConfigItemBoolean : public WUPSConfigItem {
 public:
-    static std::optional<WUPSConfigItemBoolean> CreateEx(std::optional<std::string> identifier,
+    static std::optional<WUPSConfigItemBoolean> CreateEx(std::optional<std::string_view> identifier,
```

Comment:
Can you change this to `const std::optional<std::string_view>&`, for all other instances as well?

Reply:
I initially thought this might be worse codegen, but it seems more or less identical: https://godbolt.org/z/KKx3cnqbM
Will do.